series/supportedseries.go

```diff
@@ -189,13 +190,18 @@ var ubuntuSeries = map[string]seriesVersion{
 		Version: "18.10",
 	},
 	"disco": seriesVersion{
-		Version:   "19.04",
-		Supported: true,
```

Comment:
Isn't disco still supported? Or is it that it won't be supported when 2.8 is released?

Reply:
disco is supported for another 2 weeks, so may as well update this now